--- tests/utils_test	2011-01-14 15:30:27.000000000 +0100
+++ tests/utils_test.oden	2011-01-26 11:25:58.014823466 +0100
@@ -36,6 +36,14 @@
 > "$workdir"/files/group
 > "$workdir"/files/gshadow
 
+# Ugly non-portable hacks
+LD_LIBRARY_PATH=$(pwd)/lib/.libs
+export LD_LIBRARY_PATH
+
+# it can't short circuit really good...
+LD_PRELOAD=$(pwd)/lib/.libs/libuser.so.1
+export LD_PRELOAD
+
 # Set up the client
 LIBUSER_CONF=$workdir/libuser.conf
 export LIBUSER_CONF
